infrastructure/environments/setup-environment.ts

```diff
@@ -373,6 +373,19 @@ const sshKeyQuestions = [
   }
 ]
 
+const countryQuestions = [
+  {
+    name: 'country',
+    type: 'text' as const,
+    message:
+      'What is the ISO 3166-1 alpha-3 country-code? (e.g. "NZL" for New Zealand) Reference: https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3',
+    valueType: 'VARIABLE' as const,
+    valueLabel: 'COUNTRY',
+    initial: process.env.COUNTRY,
+    scope: 'REPOSITORY' as const
+  }
+]
```

Comment:
This section could have other country-specific questions in the future. I didn't find a good existing section for it